packages/contracts-bedrock/test/L1/OptimismPortal2.t.sol

```diff
@@ -1328,6 +1340,8 @@ contract OptimismPortal2_FinalizeWithdrawal_Test is CommonTest {
     )
         external
     {
+        skipIfForkTest("Skipping on forked tests because of the L2ToL1MessageParser call below");
+
```

Comment:
This makes sense to do, because on fork tests we skip deploying the 'fake' L2 system which is deployed to the L1 [here](https://github.com/ethereum-optimism/optimism/blob/c4fcad02eb9757ebefd9594e3c953d20c9d5d44c/packages/contracts-bedrock/test/setup/Setup.sol#L257-L260).